index.js

```diff
@@ -19,6 +19,7 @@ module.exports = function (archive, target, opts, cb) {
     opts = {}
   }
   opts = opts || {}
+  opts.watch = opts.watch || opts.live
```

Comment:
this should be `var watch = opts.watch || opts.live`, in order not to create potential side effects on the opts object